doc/glossary.rst

```diff
@@ -26,3 +26,106 @@ Glossary
    Count
         The token property ``count`` in privacyIDEA is used to calculate the :term:`OTP` value
         using the HMAC-type algorithms HOTP or TOTP.
+
+   Time Step
+        The amount of time in which each totp is valid is called a timestep. Possible values are
```

Comment:
This is not quite correct and often mistaken. A TOTP token can have a timestep of 30 secs. It can still be used, if this 30 seconds are over. Technically the timestep is the divider by which the seconds since 1.1.1970 (unix system time) are devided to calculate the OTP value.
TOTP is completely equal to HOTP, the `unix system time /30` equals the `counter` in the HOTP token.

the `timestep` is different to the `timewindow` (in which the given OTP value is valid for authentication. (completely similar to the counter and window of HOTP tokens.